subworkflows/local/utils_nfcore_mag_pipeline/main.nf

```diff
@@ -80,7 +80,7 @@ workflow PIPELINE_INITIALISATION {
     ch_raw_short_reads = ch_samplesheet.map { meta, sr1, sr2, _lr ->
         meta.run = meta.run == [] ? "0" : meta.run
         meta.single_end = params.single_end
-
+        meta.sr_platform = meta.sr_platform == [] ? "ILLUMINA" : meta.sr_platform
```

Comment:
You can add the "default" key to the samplesheet schema - see https://nextflow-io.github.io/nf-schema/latest/nextflow_schema/nextflow_schema_specification/#default and https://nextflow-io.github.io/nf-schema/latest/nextflow_schema/sample_sheet_schema_examples/#nf-schema-test-case.